production/helm/loki/values.yaml

```diff
@@ -2313,6 +2315,8 @@ indexGateway:
   # -- Set the optional grpc service protocol. Ex: "grpc", "http2" or "https"
   appProtocol:
     grpc: ""
+  # -- UpdateStrategy for the StatefulSet. One of 'OnDelete' or 'RollingUpdate'.
+  updateStrategy: RollingUpdate
```

Comment:
ditto: after making the changes to make the updateStrategy configurable, in case setting `partition: 0` is different than the default, can you add it to this values.yaml, to not make a breaking change?